src/contracts/misc/interfaces/IGhoBucketCapacitySteward.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+/**
+ * @title IGhoBucketCapacitySteward
+ * @author Aave Labs
+ * @notice Defines the basic interface of the GhoBucketCapacitySteward
+ */
+interface IGhoBucketCapacitySteward {
+  /**
+   * @notice Adds/Removes controlled facilitators
+   * @dev Only callable by owner
+   * @param facilitatorList A list of facilitators addresses to add to control
+   * @param approve True to add as controlled facilitators, false to remove
+   */
+  function setControlledFacilitator(address[] memory facilitatorList, bool approve) external;
```

Comment:
Please ensure functions order is respected across files

Reply:
Done